docs/recipes/install/common/openchami_client_install.tex

```diff
@@ -0,0 +1,12 @@
+% -*- mode: latex; fill-column: 120; -*-
+\OpenCHAMI{} is very configurable, and it has a client that allows you do most of the configuration via the command line, so installing that client makes management much easier.
+% begin_ohpc_run
+% ohpc_comment_header OpenCHAMI Client Install \ref{sec:openchami_client_install}
+\begin{lstlisting}[language=bash,keywords={}]
+[sms](*\#*) latest_release_url=$(curl -s https://api.github.com/repos/OpenCHAMI/ochami/releases/latest | jq -r '.assets[] | select(.name | endswith("amd64.rpm")) | .browser_download_url')
+[sms](*\#*) curl -L "${latest_release_url}" -o ochami.rpm
+[sms](*\#*) dnf install -y ./ochami.rpm
```

Comment:
We have a macro which should be used instead of directly calling `dnf`. As we are also running on non-dnf distributions like opensuse leap.